react/SelectBox/SelectBox.jsx

```diff
@@ -86,7 +87,10 @@ const Option = ({
         className={styles['select-option__checkbox']}
       />
     )}
-    <span className={styles['select-option__label']}>{children}</span>
+    <span className={styles['select-option__label']}>
+      <span className="u-ellipsis">{customValue ? customValue : children}</span>
+      {customValue ? children : false}
+    </span>
```

Comment:
A note on this bit as it's maybe not intuitive.

For regular situations, we just put the option label in the ellipsis container, no big deal.

But when appending actions to an option with the `ActionsOption` component, the normal behaviour would mean the buttons end up in the ellipsis container, which we don't want. So in that case we can explicitly provide the option's label, and the rest of the option is rendered outside the ellipsis container.

Reply:
`customValue` dans ce cas représente le label c'est bien ça ? Dans ce cas là on peut renommer customValue en label ?

Reply:
C'est bien ca, mais la prop `label` est déja prise — c'est littéralement le `label` sous forme de `string`, alors que `customValue` est le label sous forme de composant (c'est un `ValueContainer ` de react-select ou un truc dans le genre.)

`value` est déja pris aussi — mais on peut mettre `customLabel` si c'est plus clair?

Reply:
> alors que customValue est le label sous forme de composant

labelComponent ? labelChildren ?

Reply:
I like `labelComponent` ! Let's go for that 👍